### ISSUE TITLE
In "Create A Jio", offer an option to "climb with me, I have passes"

### ISSUE THREAD

**Issue by wgao19** [climbjios-sg/climbjios-app#238]
**Is your feature request related to a problem? Please describe.**

This is similar to "sell", but less implicit on the selling but more notably an open invitation to climb together at certain gym.

The key product difference is that the climber who creates this jio do not need to specify a number of passes available, as they are willing to buy a new set of passes if number of climbers exceed the number of passes they have on hands.

**Describe the solution you'd like**

In "Create A Jio" -> "Are you looking to buy or sell passes", add an option to say "Climb with me, I have passes", with price.

When displaying jios, show "I have passes" or display unlimited passes with price.

**Describe alternatives you've considered**

**Additional context**

**Comment by therizhao** [comment]
Makes sense to display the "I have passes" in a Jiocard. Just wondering if the Create a Jio form will be confusing, now that we have two questions that controls whether the user wants passes.

Can highlight in the discord product plus design channel and see why the designers think.

**Comment by wgao19** [comment]
@therizhao thanks for considering this. I also have concerns about introducing subtly different options. With a bit more thoughts, here's what I have in mind:

For the short term, I think it's worth it to try out this option by adding an "infinite" option under "Sell" quantity. This is to make this a legit option and see how many people will start to use it.

In the long run, I feel pass trades and jios should be separate features. Jios is more time-sensitive (e.g., Wei will be climbing at Kinetics tmr 6pm onwards), whereas pass trades can be long-standing as some gyms allow people to use others' passes without the pass owner being present. For this difference in direction, I imagine the Jios bulletin to be a stream of time - gym - who's, they can form clusters, and people can tag themselves. And the pass trading listing to be more-or-less organized by gyms with filters and stuff.

With this long-term product vision, if we like it, I think "I have passes" will be an option when someone creates a jio, and we can render like a ring around the avatar of the folks who have passes so joiners know who to look for.
